ics_space_amplification_goal_test.py

```diff
@@ -145,7 +150,7 @@ def test_ics_space_amplification_goal(self):  # pylint: disable=too-many-locals
         self.wait_no_compactions_running()
 
         stress_cmd = self.params.get('stress_cmd')
-        sag_testing_values = [None, '1.5', '1.2', '1.5', None]
+        sag_testing_values = ['1.5', '1.2', '1.5', None]
```

Comment:
based on results show in https://github.com/scylladb/scylla-cluster-tests/pull/6069#issuecomment-1566174012 , it seems like what @raphaelsc commented https://github.com/scylladb/scylla-cluster-tests/pull/6069#issuecomment-1557448046 , `The lower the SAG the higher the compaction aggressiveness but also the lower the disk space usage` is not what we are getting... it most likely because we are not keeping up with the write amplification due to high write and compaction is falling behind... is that correct, @raphaelsc ?

1st wave that is `1.5` gets up to ~1TB, but 2nd wave that is `1.2` is above of 2TB of `used bytes per instance`

Reply:
@fgelcer , the first peak is the initial write of the prepare step.